APISamples/powershell/powershellSamples.ps1

```diff
@@ -33,16 +33,12 @@ Invoke-WebRequest -Uri $catalogItemsUri -Method Post -Body $payload -ContentType
 
 
 Write-Host "Download an item..."
-$downloadPath = 'C:\download\test.rdl'
-$catalogItemsApi = $ReportPortalUri + "/api/v2.0/CatalogItems(Path='/test')/Content/$value"
-$url = [string]::Format($catalogItemsApi, $item)
+$downloadPath = 'test.rdl'
```

Comment:
If the 'test.rdl' is going to be included in the repo (as mentioned in the previous comment) then the downloaded item should have a different name